scanapi/templates/html.jinja

```diff
@@ -205,6 +205,9 @@
         color: white;
         font-size: 14px;
         line-height: 20px;
```

Comment:
```suggestion
        line-height: 19px;
```

This is how it is rendering with your changes and line height 20px:

![image](https://user-images.githubusercontent.com/2728804/85860388-1d6e4f00-b795-11ea-819e-715ef1dc5719.png)

I tried to reduce 1px of the line height and this is the result:

![image](https://user-images.githubusercontent.com/2728804/85861259-6d014a80-b796-11ea-9866-f55dd597fcf3.png)

What do you think?

Reply:
I don't know, with 20px it seems more aligned with the center of the arrow icon. It would be nice to hear other opinions on this.

Reply:
True. 20 seems to better align with the arrow and 19/18 with the status code.

Anyways - I think this is good 👍